corehq/apps/users/views/mobile/users.py

```diff
@@ -1685,6 +1687,25 @@ def link_connectid_user(request, domain):
         return HttpResponse()
 
 
+@csrf_exempt
+@login_or_basic_ex(allow_cc_users=True)
```

Comment:
This needs to allow auth via connectid token, which this decorator can't do. We might need a new decorator that looks up the user via the connectid token (and validates it with connectid), or to just do it in the method.